src/psyclone/psyir/nodes/call.py

```diff
@@ -302,6 +302,17 @@ def routine(self):
         '''
         return self._routine
 
+    @property
+    def is_elemental(self):
+        '''
+        :returns: whether the routine being called is elemental (provided with\
+            an input array it will apply the operation individually to each of\
+            the array elements and return an array with the results). If this \
+            information is not known then it returns None.
+        :rtype: NoneType | bool
+        '''
+        return self._routine.is_elemental
+
```

Comment:
Do we need a equivalent property for the is_pure?

Reply:
Yes, added now.